examples/models/ds000003/models/model-001_smdl.json

```
@@ -51,23 +55,32 @@
             "trial_type.pseudoword"
           ],
           "Weights": [0.5, 0.5],
-          "Type": "t"
+          "Test": "t"
         }
       ]
     },
     {
       "Level": "subject",
+      "Name": "subject",
+      "GroupBy": ["subject", "contrast"],
+      "Model": {
+        "X": [1],
+        "Type": "meta"
+      },
+      "DummyContrasts": {}
+    },
+    {
+      "Level": "dataset",
+      "Name": "t-test",
+      "GroupBy": ["contrast"],
       "DummyContrasts": {
-        "Type": "FEMA"
+        "Test": "t"
       }
     },
     {
       "Level": "dataset",
-      "DummyContrasts": {
-        "Conditions": ["word_gt_pseudo", "task_vs_baseline"],
-        "Type": "t"
-        },
-      "Contrasts":[
+      "Name": "F-test",
```

Model and GroupBy Fields are missing:

```suggestion
      "Name": "F-test",
      "GroupBy":[
            "contrast"
      ],
      "Model":{
          "X":[
              1
          ],
          "Type":"glm"
      },
```
although I am not sure if this is the correct model.